src/solvers/smt2/smt2_conv.h

```diff
@@ -31,6 +31,22 @@ class constant_exprt;
 class index_exprt;
 class member_exprt;
 
+// TODO: This should be replaced on the spot with the
+//  smt-switch ops.h class
```

Comment:
Why though? We should just return SMT switch `Term`, which doesn't seem to reference ops? For what we're doing with these dummy terms I don't think we need anything more complicated than rose-tree of string, or even just string if it comes down to it.